doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1091 @@
+
+IDE support
+***********
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.0.0.
+SEGGER Embedded Studio Nordic Edition is no longer tested and recommended for new projects.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.0.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.0.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.0.0`_ for the list of issues valid for the latest release.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Application development
+=======================
+
+* Fixed an issue with |NCS| Toolchain delimiter handling on MacOS, which could in special situations result in the build system not being able to properly locate the correct program needed.
+
+Boards
+======
+
+* Added necessary reset functionality for using the nRF52840 SoC on the Thingy:91 as a Bluetooth controller.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Bluetooth mesh
+--------------
+
+* Added :c:struct:`bt_mesh_sensor_srv` context to relevant callbacks and APIs to help resolve the associated sensor model instance.
+  For details, see `Bluetooth mesh samples`_ and `Bluetooth libraries and services`_.
+
+See `Bluetooth mesh samples`_ for the list of changes for the Bluetooth mesh samples.
+
+Matter
+------
+
+* The CHIP Tool controller is now the recommended controller for Matter.
+  The documentation about :ref:`ug_matter_configuring_controller` has been updated accordingly.
+  For more information about the CHIP Tool controller, read the :doc:`matter:chip_tool_guide` page in the Matter documentation.
+* Added the :ref:`matter_window_covering_sample` sample, based on the Matter upstream sample.
+
+See `Matter samples`_ for the list of changes for the Matter samples.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``25e241ebcbf11b1f63dbe25546b1f10219866ad0``.
```

Comment:
The commit has to be verified.

Reply:
@LuDuda  please verify the Matter commit

Reply:
This is correct.

Reply:
thanks